src/components/selections/attributes.js

```diff
@@ -53,7 +53,28 @@ module.exports = overrideAll(templatedArray('selection', {
         valType: 'any',
         description: 'Sets the selection\'s end y position.'
     },
-
+    x_shift: {
```

Comment:
Let's drop the underscore from the name of the attribute.
We do have attributes e.g. `plot_bgcolor` and `paper_bgcolor`; but we don't want to add more.